src/Datadog.Trace.ClrProfiler.Managed.Core/AssemblyInfo.cs

```diff
@@ -1,6 +1,6 @@
 using System.Runtime.CompilerServices;
 
 [assembly: System.Reflection.AssemblyVersionAttribute("1.0.0.0")]
-[assembly: InternalsVisibleTo("Datadog.Trace.ClrProfiler.Managed, PublicKey=002400000480000094000000060200000024000052534131000400000100010025b855c8bc41b1d47e777fc247392999ca6f553cdb030fac8e3bd010171ded9982540d988553935f44f7dd58cb4b17fbb92653d5c2dc5112696886665b317c6f92795bf64beab2405c501c8a30cb1b31b1541ed66e27d9823169ec2815b00ceeeecc8d5a1bf43db67d2961a3e9bea1397f043ec07491709649252f5565b756c5")]
+[assembly: InternalsVisibleTo("OpenTelemetry.Instrumentation.ClrProfiler.Managed, PublicKey=002400000480000094000000060200000024000052534131000400000100010025b855c8bc41b1d47e777fc247392999ca6f553cdb030fac8e3bd010171ded9982540d988553935f44f7dd58cb4b17fbb92653d5c2dc5112696886665b317c6f92795bf64beab2405c501c8a30cb1b31b1541ed66e27d9823169ec2815b00ceeeecc8d5a1bf43db67d2961a3e9bea1397f043ec07491709649252f5565b756c5")]
```

Comment:
I think we would need a new SNK and the `PublicKey` should be changed.

Reply:
This will be separate PR to keep them small :)